src/sidebar.js

```diff
@@ -219,35 +219,37 @@ const initializeRoamJSSidebarFeatures = (extensionAPI) => {
     tag: "DIV",
     className: "rm-sidebar-outline",
     callback: (d) => {
-      if (extensionAPI.settings.get("ws-go-to-page"))
-        window.roamAlphaAPI.ui.rightSidebar
-          .getWindows()
-          .filter((w) => w.type === "outline")
-          .forEach((w, order) => {
-            const pageUid = w["page-uid"];
-            if (pageUid) {
-              const linkIconContainer = document.createElement("span");
-              const h = d.getElementsByClassName("rm-title-display")[order];
```

Comment:
this was the main bug - the order should be indexing the windows not the number of titles within a single window